src/cuda/Makefile

```diff
@@ -1,111 +1,105 @@
+	$(libxcdevobjfolder)/lda_c_lp96.o $(libxcdevobjfolder)/lda_c_ml1.o $(libxcdevobjfolder)/lda_c_pk09.o $(libxcdevobjfolder)/lda_c_pw.o \
+	$(libxcdevobjfolder)/lda_c_pz.o $(libxcdevobjfolder)/lda_c_rc04.o $(libxcdevobjfolder)/lda_c_rpa.o $(libxcdevobjfolder)/lda_c_vwn_1.o \
+	$(libxcdevobjfolder)/lda_c_vwn_2.o $(libxcdevobjfolder)/lda_c_vwn_3.o $(libxcdevobjfolder)/lda_c_vwn_4.o $(libxcdevobjfolder)/lda_c_vwn.o \
+	$(libxcdevobjfolder)/lda_c_vwn_rpa.o $(libxcdevobjfolder)/lda_c_wigner.o $(libxcdevobjfolder)/lda_k_tf.o $(libxcdevobjfolder)/lda_k_zlp.o \
+	$(libxcdevobjfolder)/lda_x_2d.o $(libxcdevobjfolder)/lda_xc_1d_ehwlrg.o $(libxcdevobjfolder)/lda_xc_ksdt.o $(libxcdevobjfolder)/lda_xc_teter93.o \
+	$(libxcdevobjfolder)/lda_x.o $(libxcdevobjfolder)/lda_xc_zlp.o $(libxcdevobjfolder)/lda_x_rel.o -o $(objfolder)/gpu_xcall.o
+#	$(libxcdevobjfolder)/lda_x_erf.o $(libxcdevobjfolder)/hyb_mgga_xc_wb97mv.o $(libxcdevobjfolder)/hyb_mgga_x_dldf.o $(libxcdevobjfolder)/hyb_mgga_x_m05.o \
+	$(libxcdevobjfolder)/mgga_c_b88.o $(libxcdevobjfolder)/mgga_c_bc95.o $(libxcdevobjfolder)/mgga_c_cs.o $(libxcdevobjfolder)/mgga_c_kcis.o \
+	$(libxcdevobjfolder)/mgga_c_m05.o $(libxcdevobjfolder)/mgga_c_m06l.o $(libxcdevobjfolder)/mgga_c_m08.o $(libxcdevobjfolder)/mgga_c_pkzb.o \
+	$(libxcdevobjfolder)/mgga_c_revscan.o $(libxcdevobjfolder)/mgga_c_revtpss.o $(libxcdevobjfolder)/mgga_c_scan.o $(libxcdevobjfolder)/mgga_c_tpss.o \
+	$(libxcdevobjfolder)/mgga_c_tpssloc.o $(libxcdevobjfolder)/mgga_c_vsxc.o $(libxcdevobjfolder)/mgga_k_pc07.o $(libxcdevobjfolder)/mgga_x_br89_explicit.o \
+	$(libxcdevobjfolder)/mgga_xc_b97mv.o $(libxcdevobjfolder)/mgga_xc_b98.o $(libxcdevobjfolder)/mgga_xc_cc06.o $(libxcdevobjfolder)/mgga_xc_lp90.o \
+	$(libxcdevobjfolder)/mgga_xc_zlp.o $(libxcdevobjfolder)/mgga_x_gvt4.o $(libxcdevobjfolder)/mgga_x_gx.o $(libxcdevobjfolder)/mgga_x_lta.o \
+	$(libxcdevobjfolder)/mgga_x_m06l.o $(libxcdevobjfolder)/mgga_x_m08.o $(libxcdevobjfolder)/mgga_x_m11.o $(libxcdevobjfolder)/mgga_x_m11_l.o \
+	$(libxcdevobjfolder)/mgga_x_mbeef.o $(libxcdevobjfolder)/mgga_x_mbeefvdw.o $(libxcdevobjfolder)/mgga_x_mk00.o $(libxcdevobjfolder)/mgga_x_mn12.o \
+	$(libxcdevobjfolder)/mgga_x_ms.o $(libxcdevobjfolder)/mgga_x_mvs.o $(libxcdevobjfolder)/mgga_x_pbe_gx.o $(libxcdevobjfolder)/mgga_x_pkzb.o \
+	$(libxcdevobjfolder)/mgga_x_sa_tpss.o $(libxcdevobjfolder)/mgga_x_scan.o $(libxcdevobjfolder)/mgga_x_tau_hcth.o $(libxcdevobjfolder)/mgga_x_tm.o \
+	$(libxcdevobjfolder)/mgga_x_tpss.o $(libxcdevobjfolder)/mgga_x_vt84.o
+
+#  !---------------------------------------------------------------------!
+#  ! Cleaning targets                                                    !
+#  !---------------------------------------------------------------------!
+
+.PHONY: clean
+
 clean: 
-	-rm -f *.o 
-	
+	@-rm -f *.o 
+	@-rm -f ./CUBLAS/*.o
+	@-rm -f ./CUSOLVER/*.o
```

Comment:
These folders don't exist. I see folders with lower-case letters. 
I also do not understand why we need the files in folders `CUBLAS` and `CUSOLVER`. Shouldn't these be part of the CUDA Toolkit?

Reply:
This has been fixed. What we have in cublas and cusolver folders are wrapper functions.